docs/cli/kops_rolling-update_cluster.md

```diff
@@ -20,6 +23,8 @@ kops rolling-update cluster
 ```
       --bastion-interval duration    Time to wait between restarting bastions (default 5m0s)
       --cloudonly                    Perform rolling update without confirming progress with k8s
+      --fail-on-drain-error          The rolling-update will fail if draining a node fails. Enable with KOPS_FEATURE_FLAGS='+ValidiateAndDrainRollingUpdate'
```

Comment:
Not really.  It allows for the rolling update to 'try' to validate, but not bail.  For instance, if you have one node that is not happy or a master that is not happy, you can still do a rolling update, and attempt to validate.  `--cloudonly` turns off everything.

Do I need to wordsmith the help docs?